ChangeLog.md

```diff
@@ -1,3 +1,15 @@
+## 5.4.0 (2022-09-20)
+---------------------------------------------
+- Update dropbox-api-spec to point to more recent version (Sept 01, 2022) [#431](https://github.com/dropbox/dropbox-sdk-java/pull/431)
+- Generated stone api code is now checked into repository for greater visibility of spec changes [#418](https://github.com/dropbox/dropbox-sdk-java/pull/418)
+- Changed from `Master` -> `Main` [#424](https://github.com/dropbox/dropbox-sdk-java/pull/424)
```

Comment:
`master` -> `main`

Reply:
I moved all the 5.4 references to a separate PR